requests/utils.py

```diff
@@ -447,6 +446,29 @@ def get_encodings_from_content(content):
             xml_re.findall(content))
 
 
+def _parse_content_type_header(header):
+    """Returns content type and parameters from given header
+
+    :param header: string
+    :return: tuple containing content type and dictionary of
+         parameters
+    """
+
+    tokens = header.split(';')
+    content_type, params = tokens[0].strip(), tokens[1:]
+    params_dict = {} 
+
+    for param in params:
+        if param and not param.isspace():
```

Comment:
Question: Can we not call `param = param.strip()` before the `if param` and save a call to `param.isspace()`?